docs/_docs/permalinks.md

```diff
@@ -4,27 +4,62 @@ title: Permalinks
-the [YAML Front Matter](../frontmatter/) for each post. You’re free to choose
-one of the built-in styles to create your links or craft your own. The default
-style is `date`.
+Jekyll supports a flexible way to build the permalinks for your pages, posts, and collections. A permalink is the URL for the page, post, or collection (excluding the domain name or directory folder).
 
-Permalinks are constructed by creating a template URL where dynamic elements
-are represented by colon-prefixed keywords. For example, the default `date`
-permalink is defined according to the format `/:categories/:year/:month/:day/:title.html`.
+You construct permalinks by creating a template URL where dynamic elements are represented by colon-prefixed keywords. The default template permalink is `/:categories/:year/:month/:day/:title.html`. Each of the colon-prefixed keywords is a template variable.
 
+You’re free to construct your own permalink style using the available template variables or choose one of the built-in permalink styles (such as `date`) that automatically use a template-variable pattern.
+
+## Where to configure permalinks
+
+You can configure your site's permalinks through the [Configuration]({% link _docs/configuration.md %}) file or in the [Front Matter]({% link _docs/frontmatter.md %}) for each post, page, or collection.
+
+Setting permalink styles in your configuration file applies the setting globally in your project. You configure permalinks in your `_config.yml` file like this:
+
+```yaml
+permalink: /:categories/:year/:month/:day/:title.html
+```
+
+If you don't specify any permalink setting, Jekyll uses the above pattern as the default.
+
+The permalink can also be set using a built-in permalink style:
+
+```yaml
+permalink: date
+```
+
+`date` is the same as `:categories/:year/:month/:day/:title.html`, the default. See [Built-in Permalink Styles](#builtinpermalinkstyles) below for more options.
+
+Setting the permalink in your post, page, or collection's front matter overrides any global settings. Here's an example:
+
+```yaml
+---
+title: My page title
+permalink: /mypageurl/
+---
+```
+
+Even if your configuration file specifies the `date` style, the URL for this page would be `http://somedomain.com/mypageurl/`.
+
+{% comment %}this note needs clarification
 <div class="note info">
-  <h5>Specifying permalinks through the YAML Front Matter</h5>
-  <p>
-    Built-in permalink styles are not recognized in YAML Front Matter. So
-    <code>permalink: pretty</code> will not work, but the equivalent 
-    <code>/:categories/:year/:month/:day/:title/</code>
-    using template variables will.
-  </p>
+<h5>Specifying permalinks through the YAML Front Matter</h5>
+<p>Built-in permalink styles are not recognized in YAML Front Matter. As a result, <code>permalink: pretty</code> will not work, but the equivalent <code>/:categories/:year/:month/:day/:title/</code> using template variables will.</p>
```

Comment:
This just means you can't use template variables in the `permalink` attribute in the individual files. I think it is mostly true.